packages/framework-core/src/services/graphql/graphql-generator.ts

```diff
@@ -20,7 +21,18 @@ export class GraphQLGenerator {
   private readonly subscriptionGenerator: GraphQLSubscriptionGenerator
   private readonly typeInformer: GraphQLTypeInformer
 
-  public constructor(
+  private static singleton: GraphQLGenerator | undefined
+  
+  public static getInstance(config: BoosterConfig, logger: Logger): GraphQLGenerator {
+    this.singleton = this.singleton ?? new GraphQLGenerator(
```

Comment:
@NickSeagull I had to update your suggestion because it was always creating a new instance of the class. With this change, Singleton behavior is working now

Reply:
Lovely, thanks!!